Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp

```diff
@@ -467,8 +473,12 @@ cursor: pointer;
                                 changedTextFileAccess = "Public";
                             }
                             
-                            var content = "Are you sure you want to make this text file " + changedTextFileAccess + "?";
-                            $(this).find('.modal-body p').text(content);
+                            var titlecontent = "Update Text Accessibility"
+                            var bodycontent = "Are you sure you want to make this text file " + changedTextFileAccess + "?";
+                            
+                            $(this).find('.modal-title').html(titlecontent);
```

Comment:
This will not find the dialog you've created. It'll most likely take the first element with this class it finds.